src/io/flutter/FlutterMessages.java

```diff
@@ -29,6 +29,16 @@ public static void showError(String title, String message) {
                        NotificationType.ERROR));
   }
 
+  public static void showMessage(String title, String message) {
+    final Notification notification = new Notification(
+      FLUTTER_NOTIFICATION_GOUP_ID,
+      title,
+      message,
+      NotificationType.INFORMATION);
```

Comment:
Should we add a `type` field to the daemon protocol so we can distinguish between informational messages and error or warning messages?